{"name": "test_verify_signupPremPlanWithRefCode", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <testcases.test_premiumSubscriptionPage.TestPremiumSubscription object at 0x000001D364E6B6A0>\n\n    def test_verify_signupPremPlanWithRefCode(self):\n>       self.verify_signupPremPlanWithRefCode(readData(excel,'SignUp',30,9),readData(excel,'SignUp',30,2),readData(excel,'SignUp',30,3))\n\ntestcases\\test_premiumSubscriptionPage.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nPages\\premiumSubscriptionPage.py:245: in verify_signupPremPlanWithRefCode\n    self.click(premiumPage.subscribeNow1)\nutilities\\BaseClass.py:21: in click\n    WebDriverWait(self.driver,10).until(EC.visibility_of_element_located(by_locator)).click()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"78f1af575317b53cdb15cb44a3552c6d\")>\nmethod = <selenium.webdriver.support.expected_conditions.visibility_of_element_located object at 0x000001D364E55250>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nvenv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "start": 1625048948005, "stop": 1625048958611, "uuid": "557b879c-e820-49fb-8ef0-11ee8a97c74f", "historyId": "128eaa8296a8d29a0100d7c9eb060d7b", "testCaseId": "7b3e3a5d107c18d5b50fcb09db547a10", "fullName": "testcases.test_premiumSubscriptionPage.TestPremiumSubscription#test_verify_signupPremPlanWithRefCode", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_premiumSubscriptionPage"}, {"name": "subSuite", "value": "TestPremiumSubscription"}, {"name": "host", "value": "RPS-SBC-L-0008"}, {"name": "thread", "value": "22012-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_premiumSubscriptionPage"}]}